src/edgeworkers/ew-error.ts

```diff
@@ -2,7 +2,11 @@ import {ErrorMessage} from '../utils/http-error-message';
 
 export function handleError(err, commandId) {
     try {
-        err = JSON.parse(err);
+        if(typeof err == 'object')
+            err = err.response.data;
```

Comment:
I dont think this check is needed here. After sending the request, in case of error we are converting to string `reject(cliUtils.toJsonPretty(errorObj));` and rejecting it.

Reply:
done